src/object/object_representation.c

```diff
@@ -309,14 +317,41 @@ or_class_name (RECDES * record)
   len = (int) *((unsigned char *) start);
   if (len != 0xFF)
     {
-      name = start + 1;
+      *string = start + 1;
     }
   else
     {
-      name = start + 1 + OR_INT_SIZE;
+      OR_BUF_INIT (buffer, start, -1);
+
+      rc = or_get_varchar_compression_lengths (&buffer, &compressed_length, &decompressed_length);
```

Comment:
If the string is not compressed, compresed_length and decompressed_length are set as 0 and 0xff respectively. It may depend on the "enable_string_compression" parameter.

Reply:
I've understood that `or_get_varchar_compression_lengths` works well regardless of "enable_string_compression". I'm going to resolve it.

Reply:
Thank you.